base_optional_quick_create/model.py

```diff
@@ -50,14 +48,18 @@ def _register_hook(self, cr, ids=None):
                     model_obj.check_quick_create = True
         return True
 
+    def _register_hook(self, cr):
```

Comment:
I think @Yajo means to replace the contents of _patch_quick_create with a call to _patch_method. It basically does the same thing. You'd still trigger _patch_quick_create from _register_hook.